### PR TITLE
Feature: post display

### PR DESCRIPTION
**Pull request by jcayabyab**
New version needs to `npm i` again to install a new font-awesome library. Please run it and navigate to `/category/all` to see the changes. Please give me feedback on the UI.

### CODE REVIEW

File: src/config/client.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 let config = {
   api: {
     protocol: "http",
-    host: "localhost",
+    host: "192.168.0.24",
```
Comment (harsohailB) [inline src/config/client.js]: Revert this

**Review by tylerlam1 — APPROVED** [review]
looks good

**Comment by harsohailB** [comment]
- Looks good to me. UI looks very nice!
- You may need to change data access to the posts json because of the author change (nested author) but you probably know that better than I do

**Comment by jcayabyab** [comment]
take another look when you have time

**Comment by harsohailB** [comment]
Incredible! Looks 🔥

**Pull request merged**